datahub/opal.py

```diff
@@ -0,0 +1,107 @@
+"""This module defines the data structures for the Opal model."""
+
+import pandas as pd
+
```

Comment:
Setting this as a constant at the top of the file

```suggestion

OPAL_START_DATE = "2035-01-22 00:00"

```